pkgs/servers/hasura/default.nix

```diff
@@ -25,30 +24,8 @@ let
         };
       };
 
-      # internal dependencies, non published on hackage (find revisions in cabal.project file)
-      # cabal2nix --revision <rev> https://github.com/hasura/ci-info-hs.git
-      ci-info = self.callPackage ./ci-info.nix { };
-      # cabal2nix --revision <rev> https://github.com/hasura/graphql-parser-hs.git
-      graphql-parser = self.callPackage ./graphql-parser.nix { };
-      # cabal2nix --revision <rev> https://github.com/hasura/pg-client-hs.git
-      pg-client = self.callPackage ./pg-client.nix { };
-
-      # version constrained dependencies, without these hasura will not build,
-      # find versions in graphql-engine.cabal
-      # cabal2nix cabal://dependent-map-0.2.4.0
-      dependent-map = self.callPackage ./dependent-map.nix { };
-      # cabal2nix cabal://dependent-sum-0.4
-      dependent-sum = self.callPackage ./dependent-sum.nix { };
-      # cabal2nix cabal://these-0.7.6
-      these = doJailbreak (self.callPackage ./these.nix { });
-      # cabal2nix cabal://immortal-0.2.2.1
-      immortal = self.callPackage ./immortal.nix { };
-      # cabal2nix cabal://network-uri-2.6.1.0
-      network-uri = self.callPackage ./network-uri.nix { };
-      # cabal2nix cabal://ghc-heap-view-0.6.0
-      ghc-heap-view = disableLibraryProfiling (self.callPackage ./ghc-heap-view.nix { });
-
-      # unmark broewn packages and do required modifications
+      # unmark broken packages and do required modifications
```

Comment:
These overrides are much better suited for the `pkgs/development/haskell-modules/configuration-common.nix` file.

Also the use of `unmarkBroken` is not the preferred approach here. If those packages actually compile, they should be marked unbroken by removing them from the broken list in `configuration-hackage2nix.yaml` so that everyone using nixpkgs can benefit from those packages.